requirements.txt

```diff
@@ -4,3 +4,4 @@
 
 Adafruit-Blinka
 adafruit-circuitpython-busdevice
+adafruit-circuitpython-typing
```

Comment:
@tekktrik is this required? I don't see it in any other library but not sure myself.

Reply:
Mark, it’s there because of the import in the try block of Readable and Writable buffer

Reply:
Yup, the buffer classes come from that.  Technically it comes with Blinka at this point I believe, but better to be explicit.